src/components/app-rating-mobile/AppRatingMobile.tsx

```diff
@@ -9,18 +9,41 @@ import { styles } from '~/components/app-rating-mobile/AppRatingMobile.styles'
 
 interface AppRatingMobileProps extends RatingProps {
   reviewsCount: number
+  linkHash?: ReactElement
+  style?: {
+    variantOption?: string
+    fontSize?: string
+    starMobile?: object
+    reviews?: object
+    rating?: object
+  }
 }
 
-const AppRatingMobile: FC<AppRatingMobileProps> = ({ value, reviewsCount }) => {
+const AppRatingMobile: FC<AppRatingMobileProps> = ({
+  value,
+  reviewsCount,
+  linkHash,
+  style
+}) => {
   const { t } = useTranslation()
 
   return (
     <Box sx={styles.root}>
       <Box data-testid='number-box' sx={styles.number}>
-        <StarSharp data-testid='star-icon' sx={styles.starMobile} />
-        <Typography variant={'h6'}>{value}</Typography>
+        <StarSharp
+          data-testid='star-icon'
+          sx={style?.starMobile ? style?.starMobile : styles.starMobile}
+        />
+        <Typography sx={style?.rating} variant={'h6'}>
+          {value}
+        </Typography>
       </Box>
-      <Typography variant={'caption'}>
+      <Typography
+        component={linkHash}
+        sx={style?.reviews}
+        to={'#'}
+        variant={style?.variantOption}
```

Comment:
What is this? Variant from styles? If you pass styles, you can delete variant props